tests/utils/waitForPromisesToResolveWithAct.js

```diff
@@ -0,0 +1,21 @@
+import {act} from '@testing-library/react-native';
+import waitForPromisesToResolve from './waitForPromisesToResolve';
+
+/**
```

Comment:
I love this comment, but what's not clear to me is when it should be used. Is there some guidance that can be added for that? Like... "Use when..." and "Do not use when..."